sound/soc/sof/intel/hda-mlink.c

```diff
@@ -340,6 +340,21 @@ static void hdaml_link_set_lsdiid(u32 __iomem *lsdiid, int dev_num)
 	writel(val, lsdiid);
 }
 
+static void hdaml_shim_map_stream_ch(u16 __iomem *pcmsycm, int lchan, int hchan,
+				     int stream_id, int dir)
+{
+	u16 val;
+
+	val = readw(pcmsycm);
```

Comment:
minor nit. can be moved to the declaration line

Reply:
indeed, but the style for this file is to keep variables and first read separate.